### ISSUE TITLE
Post Proposal: Meet the Hoodies (Series)

### ISSUE THREAD

**Issue by schisepo** [hoodiehq/editorial#14]
> What is your blog post about? Describe it in about one paragraph.

This series will celebrate the Hoodies, our fantastic community of contributors. Each episode will feature one contributor and will involve them answering the below questions and submitting a quirky photo to go with the post:

1. What's your favorite animal?
2. What's your role / what have you contributed to Hoodie? 
3. What's your favorite use of Hoodie in the wild?
4. What in your opinion makes the Hoodie project unique and interesting?
5. What advice do you have for aspiring Hoodie contributors?

> Approximately how long do you think the post will be?

Each post will be a maximum of 1000 words depending on how verbose the contributor is :smile: 

> Approximately how long do you think it will take you to write this post?

2 days for each contributor

> Do you think your post could make use of any illustrations or photographs?

Yes! Photographs of the featured Hoodie and an animal muse would be cool :grin: 

> Do you have any links to previous writing?

No.

> Where else can we find you on the internet? (e.g. your blog, Tumblr, Medium, Twitter, etc.)

https://twitter.com/schisepo

> Anything else you want to tell us?

This will be a fantastic series :rocket:

**Comment by markpalfreeman** [comment]
Thanks for starting this!

Other question ideas to consider:
- What's your role / what have you contributed to Hoodie? *(maybe this is the same as no. 3, but it sounds like it could be for side projects)*
- What's your favorite use of Hoodie you've seen in the wild?

(I think from a process perspective, this Meet the Hoodies idea is already "approved" via #9, so maybe we should move forward with creating an Interview Questions .md document that can be edited/refined—I'm still getting used to project management in GitHub 😁)

**Comment by schisepo** [comment]
Thanks for the feedback @markpalfreeman :smile: 

Yes, the idea is approved so this was mainly to test-drive the process (I'm pretty new to git-hub as well so its a learning curve) I'm thinking that we consider the issue approved and comment on here with other question ideas (Like your fantastic contribution above :smile_cat: ) or  I can create a sub-folder for proposals like @remixz has done for documents and move questions to a file in that that folder.

**Comment by remixz** [comment]
:+1: for test-driving the process now. The more time we spend with our processes early, the better prepared we'll be to use them efficiently later with more people, as we take on more editorial projects.

For these kind of recurring series, I think it'd make sense that we spend a couple days collecting feedback from everyone in the community, and then documenting what we came up with in the repo, as @schisepo has suggested.

By the way, all these question ideas look great! :tada: It'd be great to have a balance of Hoodie-specific questions and general open-source questions, so that these can become valuable resources for people wanting to get started in the open-source community.

---

Now, for some thoughts on what we do after we've determined the questions:

* We'll have to find someone who wants to be interviewed. I have a feeling this won't be too hard. :wink:
* We should use this repository to do the interviewing. It could be pretty easy as well: We'll create a new issue, with a title like "Meet the Hoodie - @[GitHub username]". That issue will serve as the main point of contact for everyone involved with that interview. 
* We should post the questions in the issue body, and @-mention the person being interviewed, asking them to answer the questions when they can. We should give the interviewee an overall guideline on how long their answers should be.
* Once they've answered, we can compile the answers into a Markdown file, and PR this repository, as per our process. Any editors can make tweaks as needed, whether it be simple grammar issues, or adding additional context to an answer, as needed.
* From there, it's pretty much just [the blog post process](https://github.com/hoodiehq/editorial/blob/master/documents/blog-process.md) of editing, approving, PRing the hood.ie repository, and posting at a scheduled time. (We'll need to figure out what day of week/time to post these interviews.)
* For our first Meet the Hoodie, I think it'd be cool to link to the issue made for the interview, so we can show people what we're doing behind the scenes, as a completely open editorial team. We might be able to get some more people involved, as well!

**Comment by Charlotteis** [comment]
Rather than dogs/cats I'd prefer "whats your favourite animal?" :)

**Comment by schisepo** [comment]
@markpalfreeman @Charlotteis, @remixz  looks like the 72 hour review window is up for this proposal. :tada:  Going with @remixz suggestion, I'd like to create an issue for our first featured Hoodie to respond to the interview questions, after which we'll create a file in markdown  for editing and subsequent publishing :smile: 

@Charlotteis - who are the Hoodies :see_no_evil: :blush: ? i.e who are the candidates for our series and is there somewhere I can get a list? I suggest featuring them in ascending (or descending) alphabetical order of name so that the order in which they appear does not imply their value to the project.

**Comment by Charlotteis** [comment]
@schisepo The 72 hour window for was process changes (e.g. We should only refer to each other by secret codename), not proposals :) However, I see no reason why we cannot go ahead with this and making a new issue for the first Hoodie would be great.

@janl could you help us with the 'Hoodies'? I know we've already interviewed some in the past. I also know I have not been interviewed, nor has @hipsterbrown but if we could work together to compile a list that'd be great.

**Comment by HipsterBrown** [comment]
There is a decent list here, http://hood.ie/community/, although it may not be the most up-to-date/complete list.

**Comment by Charlotteis** [comment]
not up to date (in the sense that I think some people have left?) but that is a good start. Need to work out who has been interviewed (the blog should be able to tell you that if you go through the archives) unless we wish to interview everyone again :)

**Comment by schisepo** [comment]
thanks @HipsterBrown and @Charlotteis Here's the list :point_down: I've put ** next to names featured in the "A Hoodie for you" series. We'll start with @espy 's profile :smiley: 

Alex Feyerke	espy
Caolan McMahon	caolan
Carolina Buzio	carolinabuzio
Gregor Martynus	gr2m
Jan Lehnardt	janl
Jan-Erik Rediger	
Julia Schmidt	
Katrin Apel	kaalita
Kevin Lorenz	verpixelt
Lena Reinard	
**Lewis Cowper	lewiscowper
Ola Gasidlo	zoepage
Stephan Bonnemann	boenneman
Sven Lito	svnlto
**Elmar Burke	
**Ray Daly

**Comment by Charlotteis** [comment]
you left me (Charlotte Spencer) off :) And some of our other hoodies. The folks under "contributors" are also, as far as I am aware, considered part of our core team they just aren't necessarily around 24/7. Though I think lately I and @HipsterBrown seem to be :)

**Comment by schisepo** [comment]
Thanks @Charlotteis, will amend list to include everyone mentioned as a contributor. Are there any people on this list + contributors who have since left?

**Comment by Charlotteis** [comment]
The answer is "probbably" but I think Jan, Gregor etc. will have more info on who has left. For example I don't know if I've spoken to or seen half of the contributor people contribute whilst I've been around, but I'm unsure.

**Comment by schisepo** [comment]
@janl @gr2m when you get a moment, please parse the above list :point_up: for inactive/resigned Hoodie contributors who will not be featured in our series.

**Comment by gr2m** [comment]
We already had "meet the Hoodies" with @zoepage [link](http://hood.ie/blog/meet-the-hoodies-episode-2-ola.html) and @svnlto [link](http://hood.ie/blog/meet-the-hoodies-episode-1-sven.html)

Currently inactive are

- Caolan McMahon
- Carolina Buzio
- Jan-Erik Rediger
- Julia Schmidt
- Katrin Apel
- Lena Reinard

On the other side, we added a _ton_ of new people to the Hoodie Contributors and we really missed out to celebrate that :) Like all the great editorial people of course. On the code side we could reach out to @nathanstilwell, @nickcolley and probably more :)

One feedback on the questions: "What's your favorite use of Hoodie in the wild?" I think our editorial work is as important as our work on code, do you think the question is more code-centric though? Maybe it’s just me 

Thanks so much everyone. I can’t say enough how much this excites me :)

**Comment by janl** [comment]
Heya, this all looks very fabulous. I don’t think we need to worry about coming up with a complete list right now before we continue with this. As we go along, and as everyone is more involved here, next candidates will show up organically. And if not, always feel free to reach out to me or @gr2m or @espy to suggest somebody.

**Comment by Charlotteis** [comment]
@janl are you wanting to close this, then?

**Comment by janl** [comment]
Mainly, it looked to me that the process might be stuck on not having a complete list of people to put into the series and all I wanted is unblock this :)

@Charlotteis if this isn’t needed for anything else, we should close it, yes,

**Comment by Charlotteis** [comment]
Right, then let's close it. :tada:
